pgjdbc/src/main/java/org/postgresql/core/v3/DefaultSqlSerializationContext.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2025, PostgreSQL Global Development Group
+ * See the LICENSE file in the project root for more information.
+ */
+
+package org.postgresql.core.v3;
+
+/**
+ * Provides a default implementation for {@link SqlSerializationContext}.
+ */
+enum DefaultSqlSerializationContext implements SqlSerializationContext {
+  STDSTR_IDEMPOTENT(true, true),
```

Comment:
I think it would be useful to document why and where each of these are used.
For instance we use IDEMPOTENT to send the query to the backend
STD is standard conforming strings which is a setting in the backend.

Reply:
I've added the doc to the enum entries